### DISCUSSION TITLE
What is the unit of `pred_time_val ` and `pred_time_val_marginal ` in leaderboard?

### DISCUSSION THREAD

**Discussion by ryangawei** [autogluon/autogluon#4715] [Q&A]
Hi, I want to know what time unit of the two metrics are, just to be sure

**Comment by LennartPurucker** [comment]
Heyho, I think the details are:

* `pred_time_val_marginal`: The time it takes to only predict with this model. 
* `pred_time_val`: The time it takes to get predictions for this model. This includes the time it takes to get the predictions from base models (e.g., in stacking). Thus, a model could have a very large `pred_time_val`, but its contribution to the sum of the prediction time is only `pred_time_val_marginal`. 

See https://auto.gluon.ai/stable/api/autogluon.tabular.TabularPredictor.leaderboard.html for more details.
